MembraneVideoroomDemo.xcodeproj/project.pbxproj

```diff
@@ -511,7 +511,7 @@
 			buildSettings = {
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = YGRG6VYGY2;
```

Comment:
please don't change the dev team, it's SWM team
if you aren't in SWM team let me know, I'll add you